packages/gatsby-source-drupal/src/__tests__/index.js

```diff
@@ -213,6 +213,13 @@ describe(`gatsby-source-drupal`, () => {
     )
   })
 
+  it(`Skips File Downloads`, async () => {
+    const skipFileDownloads = true
+    expect(createRemoteFileNode).toBeCalledTimes(8)
+    await sourceNodes(args, { baseUrl, skipFileDownloads })
+    expect(createRemoteFileNode).toBeCalledTimes(8)
+  })
```

Comment:
I'm going to leave this up to you. But considering there was a logic error before, how did this test still pass? Or is there a test missing?

Reply:
There is a test missing. There are two paths through the code. The first is on initial build, the second is when a webhookBody is passed in (through a content update/incremental build).

I'm going to write another test to test content updates that contain a file. Thanks for catching this! I will ping you when I commit up the new test.